.github/workflows/ci.yml

```diff
@@ -0,0 +1,11 @@
+on: push
```

Comment:
Maybe add `workflow_dispatch:` here so that it can be started manually?
```suggestion
on:
  push:
  workflow_dispatch:
```

Reply:
Probably doesn't hurt :)